ep/build-velox/src/get_velox.sh

```diff
@@ -17,7 +17,7 @@
 set -exu
 
 VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=2023_12_29
+VELOX_BRANCH=2024_01_02
```

Comment:
can you please switch back to use the `update` branch? this is b/c some other people are actually using this branch also

thanks, -yuan